docs/agent/version-support-policy.mdx

```diff
@@ -44,10 +44,7 @@ Extended support beyond LTS is available to customers on a commercial basis. It
 
 | Support Status | Release | Release Date | Latest | Active Support | End of Life |
 | -------------- | ------- | ------------ | ------ | -------------- | ----------- |
-| Active         | 3.18x   | 2024-11-07   | 3.18.3 | 2025-11-07     | 2026-01-07  |
-| Active         | 3.18x   | 2024-10-31   | 3.18.2 | 2025-10-31     | 2025-12-10  |
-| Active         | 3.18x   | 2024-10-17   | 3.18.1 | 2025-10-24     | 2025-12-10  |
-| Active         | 3.18x   | 2024-10-17   | 3.18.0 | 2025-10-17     | 2025-12-10  |
```

Comment:
These should not be removed

Reply:
Disagree - the support policy is for 3.18.x, not a separate support policy for each patch version.

Reply:
You right